Makefile

```diff
@@ -52,8 +52,13 @@ build-integration: ## Build integration test binary
 	mkdir -p bin
 	$(DOCKER_CMD) go build $(GOGCFLAGS) -o bin/integration github.com/openshift/machine-api-operator/test/integration
 
-test-e2e:
-	go run ./vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e/openshift/*.go -alsologtostderr
+test-e2e: ## Run openshift specific e2e test
+	go test -timeout 60m \
+		-v ./vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
+		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
+		-machine-api-namespace $${NAMESPACE:-openshift-machine-api} \
+		-ginkgo.v \
+		-args -v 5 -logtostderr true
```

Comment:
What's the trailing `true` for?

Reply:
It's the value for the `-logtostderr` argument.

Reply:
Isn't that implicitly `true` if specified?